components/schemes/Color/tests/unit/MDCSemanticColorSchemeTests.m

```diff
@@ -177,4 +177,23 @@ - (BOOL)compareColorsWithFloatPrecisionFirstColor:(UIColor *)firstColor
           MDCCGFloatEqual(fAlpha, sAlpha));
 }
 
+- (void)testColorSchemeCopyTest {
+  // Given
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  MDCSemanticColorScheme *colorSchemeCopy = [colorScheme copy];
```

Comment:
The `copy` is the `// When` for this test. Please add the comment.

Reply:
Done.